sympy/integrals/manualintegrate.py

```diff
@@ -971,6 +972,27 @@ def exp_rule(integral):
         return ExpRule(integrand, symbol, E, integrand.args[0])
 
 
+def powsimp_rule(integral):
+    """
+    Strategy that simplifies the exponent of a power.
+    exp(a*x**2) * exp(b*x) -> exp((a*x**2 + b*x))
+    For example, this is useful for the ErfRule.
+    """
+    integrand, symbol = integral
+    a = Wild('a', exclude=[symbol])
+    b = Wild('b', exclude=[symbol])
+    k = Wild('k', exclude=[symbol])
+
+    match = integrand.match(k**(a*symbol**2) * k**(b*symbol))
+
+    if not match:
+        return
+
+    simplified = powsimp(integrand, combine='exp')
+
+    return None if simplified == integrand else RewriteRule(integrand, symbol, simplified, integral_steps(simplified, symbol))
```

Comment:
This line is long. There is no need for it to be long though:
```python
if simplified != integrand:
    steps = integral_steps(simplified, symbol)
    return RewriteRule(integrand, symbol, simplified, steps)
```

Reply:
Done